pycode/memilio-generation/memilio/tools/config.json.txt

```diff
@@ -30,10 +30,10 @@
     {
         "source_file": "${memilio_SOURCE_DIR}/models/ode_seir/model.cpp",
         "namespace": "mio::osecirvvs::",
-        "python_module_name": "test_secirvvs",
-        "python_generation_module_path": "${memilio-python_SOURCE_DIR}",
-        "skbuild_path_to_database": "${memilio-python_BINARY_DIR}",
-        "target_folder": "${memilio-python_SOURCE_DIR}/memilio/tools",
+        "python_module_name": "osecirvvs",
+        "python_generation_module_path": "${memilio-python_SOURCE_DIR}/../memilio-generation",
+        "skbuild_path_to_database": "${memilio-python_BINARY_DIR}/../memilio-generation",
+        "target_folder": "${memilio-python_SOURCE_DIR}/../memilio-generation/memilio/tools",
```

Comment:
Why is that different for this and the other models now?

Reply:
It should resolve the problem of having one of the other python packages as source directory when building memilio-generation after them, eg. memilio-simulation. So we always go one step up into pycode and then back into the generation directory